Ginger/GingerCore/Platforms/PlatformsInfo/JavaPlatform.cs

```diff
@@ -72,25 +72,70 @@ public override ObservableList<Act> GetPlatformElementActions(ElementInfo elemen
+                    }
+                    else if (action.Equals(ActUIElement.eTableAction.GetSelectedRow) || action.Equals(ActUIElement.eTableAction.ActivateRow))
+                    {
+                        elementAction = ActUIElement.eElementAction.TableRowAction;
+                    }
 
-            if (actionList.Count > 0)
+                    var actUITableAction = new ActUIElement()
+                    {
+                        Description = action + " : " + elementInfo.ElementTitle,
+                        ElementType = eElementType.Table,
+                        ElementAction = elementAction,
+                    };
+                    actUITableAction.GetOrCreateInputParam(ActUIElement.Fields.RowSelectorRadioParam, "RowNum");
+                    actUITableAction.GetOrCreateInputParam(ActUIElement.Fields.LocateRowType, "Row Number");
+                    actUITableAction.GetOrCreateInputParam(ActUIElement.Fields.LocateRowValue, "0");
+                    actUITableAction.GetOrCreateInputParam(ActUIElement.Fields.ColSelectorValue, ActUIElement.eTableElementRunColSelectorValue.ColNum.ToString());
+                    actUITableAction.GetOrCreateInputParam(ActUIElement.Fields.LocateColTitle, "0");
+                    actUITableAction.GetOrCreateInputParam(ActUIElement.Fields.ControlAction, action.ToString());
+                    actUITableAction.GetOrCreateInputParam(ActUIElement.Fields.WaitforIdle, ActUIElement.eWaitForIdle.Medium.ToString());
+                    UIElementsActionsList.Add(actUITableAction);
+                }
+            }
+            else
             {
-                foreach (var action in actionList)
+                var actionList = GetPlatformUIElementActionsList(elementType);
+
+                if (actionList.Count > 0)
                 {
-                    UIElementsActionsList.Add(
-                        new ActUIElement()
-                        {
-                            Description = action + " : " + elementInfo.ElementTitle,
-                            ElementAction = (ActUIElement.eElementAction)action,
-                            ElementType = elementType,
-                        }
-                        );
+                    foreach (var action in actionList)
+                    {
+                        UIElementsActionsList.Add(
+                            new ActUIElement()
+                            {
+                                Description = action + " : " + elementInfo.ElementTitle,
+                                ElementAction = (ActUIElement.eElementAction)action,
+                                ElementType = elementType,
+                            }
+                            );
+                    }
                 }
             }
+            
             return UIElementsActionsList;
         }
 
+        private List<ActUIElement> CreateActUITableAction(List<ActUIElement.eTableAction> tableActionList)
+        {
```

Comment:
What is this method ? do we need it or can be removed?

Reply:
will remove this method.